data/update-2015032301.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE sites ADD site_map_url VARCHAR(2100);
+ALTER TABLE sites ADD crawl_method ENUM('CRAWL_ONLY', 'SITE_MAP_ONLY', 'HYBRID') NOT NULL DEFAULT 'HYBRID';
+ALTER TABLE scanned_page ADD found_with ENUM('SITE_MAP', 'CRAWL') NOT NULL;
```

Comment:
The field is defined as `NOT NULL`, which to me says 'you must choose between one of the two options'.  Defining a default does not make sense to me because the application layer is always required to define which one was used.